queries/synths/useBaseFeeRateQuery.ts

```diff
@@ -7,22 +7,33 @@ import { CurrencyKey } from 'constants/currency';
 import { appReadyState } from 'store/app';
 import QUERY_KEYS from 'constants/queryKeys';
 
-const useBaseFeeRateQuery = (currencyKey: CurrencyKey | null) => {
+const useBaseFeeRateQuery = (
+	sourceCurrencyKey: CurrencyKey | null,
+	destinationCurrencyKey: CurrencyKey | null
+) => {
 	const isAppReady = useRecoilValue(appReadyState);
 	const { synthetixjs } = Connector.useContainer();
 
 	return useQuery(
-		QUERY_KEYS.Synths.BaseFeeRate(currencyKey!),
+		QUERY_KEYS.Synths.BaseFeeRate(sourceCurrencyKey!, destinationCurrencyKey!),
 		async () => {
 			const { SystemSettings } = synthetixjs!.contracts;
 
-			const baseFeeRate = await SystemSettings.exchangeFeeRate(
-				ethers.utils.formatBytes32String(currencyKey as string)
+			const sourceCurrencyFeeRate = await SystemSettings.exchangeFeeRate(
+				ethers.utils.formatBytes32String(sourceCurrencyKey as string)
 			);
-			return baseFeeRate ? baseFeeRate : null;
+
+			const destinationCurrencyFeeRate = await SystemSettings.exchangeFeeRate(
+				ethers.utils.formatBytes32String(destinationCurrencyKey as string)
+			);
+
+			return sourceCurrencyFeeRate || destinationCurrencyFeeRate
+				? sourceCurrencyFeeRate.add(destinationCurrencyFeeRate)
+				: null;
```

Comment:
Do we want to check if both values exist or only one? I'm worried that there might be a case where one of `sourceCurrencyFeeRate` and `destinationCurrencyFeeRate` may be falsy, but the code will still attempt to run line 31, causing an error.

Reply:
Thanks. The change has been made.